client/flutter/lib/main.dart

```diff
@@ -86,6 +86,7 @@ class _MyAppState extends State<MyApp> {
         primaryColor: Constants.primaryColor,
         accentColor: Constants.textColor,
         brightness: Brightness.light,
+        appBarTheme: AppBarTheme(brightness: Brightness.light),
```

Comment:
This is the best way of doing it. I think the reason this happened in the first place is that the `ThemeData` is not set up optimally in general.

Reply:
Yeap. It is because in Material app bars have the primary color as background color and dark brightness by default.

I think we should improve the ThemeData and implement the desired style guidelines so the app has a consistent branding. Right now I see colors and font sizes everywhere around the app.

I would like to help in that if needed